astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -1914,9 +1914,9 @@ def check2(self, function, *args, **kwargs):
     @pytest.mark.parametrize(
         "kwargs",
         (
-            dict(return_index=True, return_inverse=True),
```

Comment:
All these test cases are very specifically written this way to make them look like arguments to functions. I really see no reason to decrease readability because of some ruff "rule".